### DISCUSSION TITLE
Reporter can not support Log4j2 mixed Sync&Async config

### DISCUSSION THREAD

**Discussion by bjhx2003** [apache/skywalking#11409] [General]
Async or  AsyncLogger  reporter wihout endpoint and traceId。
only support  -Dlog4j2.contextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector，but I need use fixed sync&async
how can i do？

**Comment by wu-sheng** [comment]
SkyWalking log reporter is already async internal.

**Reply by bjhx2003** [reply]
i use agent 9.0.0，the configuration is shown in the following figure，when i  access the dashboard，no endpoint and trace, but traceId can be printed locally
![image](https://github.com/apache/skywalking/assets/11609627/fc37f79f-f928-4638-bc99-5bfb8140da75)
![image](https://github.com/apache/skywalking/assets/11609627/0e9d2612-6754-48b4-a62a-58d7c40e545d)

**Reply by wu-sheng** [reply]
I mean, don't put skywalking log in async mode. The context added in the sync mode, and internally, skywalking log would report logs with context through async. You don't need to set up explicitly.

**Reply by bjhx2003** [reply]
That's mean i can use the sync mode without any issues. Could you please specify the package or class where the soucre code is located ？i'll takte a look and find out
